pkg/api/handlers/compat/containers_logs.go

```diff
@@ -72,18 +72,21 @@ func LogsFromContainer(w http.ResponseWriter, r *http.Request) {
 
 	var until time.Time
 	if _, found := r.URL.Query()["until"]; found {
-		// FIXME: until != since but the logs backend does not yet support until.
-		since, err = util.ParseInputTime(query.Until)
-		if err != nil {
-			utils.BadRequest(w, "until", query.Until, err)
-			return
+		if query.Until != "0" {
+			// FIXME: until != since but the logs backend does not yet support until.
```

Comment:
Can you drop the comment above?

Reply:
sorry, forgot about that. Fixed now @vrothberg